demo/Generic nonlinear delta node.ipynb

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
<p>remove empty cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/biaslab/ReactiveMP.jl/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
we should considering removing this demo